test/resources/PSR with Login/environments/LocalHost - PSR - with LOGIN.bru

```diff
@@ -0,0 +1,5 @@
+vars {
+  pension-scheme-return: http://localhost:10700/pension-scheme-return
+  auth: http://localhost:8585
+  bearer_token: BXQ3/Treo4kQCZvVcCqKPlJ4wf4lIOaO27ua3kjMs3S6iFOAsYhjaQAEtHMEd7377fZnbGSgp5WiM4DlZst2/pxfAbw+lYTjsnYfjyofDN1OhVf06Xj5KByFuciwhPWi05Xz1oCm3LdYKDST9N0epHaDkKrec9hPESB1GeXRgB79KwIkeIPK/mMlBESjue4V
```

Comment:
Is it right to have this version controlled?

Reply:
removed